tetris.py

```diff
@@ -107,8 +107,7 @@ def right_mediator(self, event):
 class TetrisGame:
     def __init__(self):
         self.landed_blocks = []
-        self.current_block = None
-        self.upcoming_block = None
+        self.upcoming_block_shape = None
```

Comment:
Defining all attributes in `__init__` makes it easy to see what attributes the class has:

```suggestion
        self.upcoming_block_shape = None
        self.upcoming_block_center = None
        self.current_block_center = None
```